test_staticjinja.py

```diff
@@ -131,9 +133,12 @@ def watch(self):
 def test_should_handle(reloader, template_path):
     template1_path = str(template_path.join("template1.html"))
     test4_path = str(template_path.join("test4.html"))
+
+    open(test4_path, 'w').close()
```

Comment:
Yeah that doesn't look great. I think you want to use the `tmpdir` fixture built into pytest. https://pytest.org/latest/tmpdir.html